app/models/models.py

```diff
@@ -122,20 +122,29 @@ class Tweet(models.Model):
         'user_screen_name' : None,
         'user_verified' : False,
         'sentiment_score' : False,
-        'imdbID': False,
+        'linkedMovies' : False,
         'tweetID': False,
     }
 
     ## Populate the current twitter object with values returned from a Twitter API request
-    def fillWithStatusObject(self, tweet, movie_title):
+    def fillWithStatusObject(self, tweet, movie):
         """
         :param tweet: an object of the class twitter.Status (returnd by Twitter API)
         :return updated_tweet: this tweet, updated with the data from the given tweet
         """
-        ## Do not create a new Tweet object for this tweet if it is invalid or already exists in our db
-        if tweet is None or not isinstance(tweet, twitter.Status) or Tweet.objects.filter(tweetID=tweet.id):
+        ## Do not create a new Tweet object for this tweet if it is invalid
+        if tweet is None or movie is None or not isinstance(tweet, twitter.Status):
             return None
 
+        existingTweets = Tweet.objects.filter(tweetID=tweet.id)
```

Comment:
Shouldn't this only ever return one tweet? If so we should use `tweet.objects.get()`